style.css

```diff
@@ -71,7 +71,156 @@ section {
+   }
+ .work-section {
+    background-color:white;
+     }
+   .placeholder {
+    width: 327px;
+    height: 237px;
+    left: 24px;
+    top: 0px;
+    right: 24px;
+   }    
+    
+ h2 {
+     font-family:"Create ROund",sans-serif;
+     font-style: normal;
+     font-weight: normal;
+     font-size: 40px;
+     color: #1C1A19;
+     width: 327px;
+     height: 83px;
+     line-height: 52px;
+     border-bottom: 1px solid #212121;
+     }
+   .mini-title {
+    font-family: Crete Round;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 32px;
+    line-height: 44px;
+    color: #091E42;
+     }  
+    .dummy-text {
+    font-family: Inter;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 24px;
+    color: #091E42;
+    }
+    .invite {
+      font-family: Inter;
+      font-style: normal;
+      font-weight: bold;
+      font-size: 17px;
+       line-height: 24px;
+       color: #FFFFFF;
+       background: #FF6B00;
+        padding: 10px;
+        text-align: center;
+    }
+    .see-project{
+        margin: 20px 0;
+        
+    }
+    .data{
+        font-family: Crete Round;
+         font-style: normal;
+         font-weight: normal;
+         font-size: 32px;
+         line-height: 44px;
+    }
+    .selections {
+        font-family: Inter;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 16px;
+        line-height: 24px;
+        width: 295px;
+        height: 87px;
+        left: 16px;
+        top: 101px;
+
+    }
+    .buttons {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 10px 12px;
+        width: 56px;
+        height: 40px
+        
+    }
+    .art-item{
+        background-image: url(icons/Img\ Placeholder\ \(1\).png);
+    }
+
+    .about-me {
+        left: 0px;
+        background: #1C1A19;
+
+    }
+    .buttons {
+        display: flex;
+        align-items: center;
+        background: rgba(255, 255, 255, 0.24);
+        gap: 5px;
+
+    }
+    .h3-title {
+        font-family: 'Crete Round',sans-serif;
+        font-style: normal;
+        font-weight: normal;
+        font-size: 75px;
+        color: #FF6B00;
+
+    }
+   button{
+    border: none;
+   }  
+
+    .css{
+    color: black;
+    border: solid 1px rgb(2, 16, 29);
+    padding: 5px 10px;
+    width: 20%;
+    text-align: center;
+    margin-right: 10px;
+    }
+    .css-container{
+    display: flex;
+    }
+    .resume {
+        background: #FF6B00;
+        padding: 7px;
+        color: white;
+        margin-bottom: 88px;
+    }
+
+    .about-paragraph {
+        color: #F4F5F7;
+        font-family: "Inter", sans-serif;
+        font-size: 13px;
+    }
+    .work-on {
+        display:flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        width: 100%;
+        list-style: none;
+    }
+    .list {
+        list-style: none;
+    }
```

Comment:
- [ ] [Required] We can use flexbox to style the lists. This will align your project with the requirements.